### PR TITLE
packit: add support to EPEL10

### PR DESCRIPTION
**Pull request by dougsland**
Add support to EPEL10 releases

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -134,3 +136,4 @@ jobs:
       - fedora-39
       - fedora-38
```
Comment (sandrobonazzola) [inline .packit.yaml:137]: worth to be done in its own patch, but you can drop fedora-38, it has gone EOL long time ago.

**Review by sandrobonazzola — APPROVED** [review]
LGTM

File: .packit.yaml (hunk 1 above)
Comment (dougsland) [reply .packit.yaml:137]: Agreed. It's in my TODO to do so. I was even thinking in remove fedora-39 and added 41 and 42. WDYT @sandrobonazzola  too early?

Comment (sandrobonazzola) [reply .packit.yaml:137]: Let's wait another couple of months for that.

File: .packit.yaml [hunk 2]
```diff
@@ -116,6 +116,7 @@ jobs:
...
       - epel-9
+      - epel-10
```
Comment (lsm5) [inline .packit.yaml:119]: I suspect packit will complain by filing an upstream issue until epel-10 is released. But feel free to try this out if you like. LGTM with that caveat.

**Comment by dougsland** [comment]
CI/CD is related to the issue we are debugging in github actions:
```
Writing manifest to image destination
Error: OCI runtime error: crun: the requested cgroup controller `pids` is not available
Error: ffi-qm container failed to start. Status: created
```
